code/game/objects/effects/landmarks.dm

```diff
@@ -724,3 +724,16 @@ INITIALIZE_IMMEDIATE(/obj/effect/landmark/start/new_player)
 
 /obj/effect/landmark/navigate_destination/disposals
 	location = "Disposals"
+
+/obj/effect/landmark/random_room
+	name = "random room spawner"
+	icon = 'icons/effects/landmarks_static.dmi'
+	icon_state = "lift_id"
+	dir = NORTH
+	var/room_width = 0
+	var/room_height = 0
+
+/obj/effect/landmark/nodelapc
```

Comment:
ne yapıyo bu

Reply:
emptyer yerine artık apcyi kendim sildirtiyom, o tilede o landmark varsa apcyi silmiyo

Reply:
zaten apc olmasını istediğin yerde apcyi silip tekrar koymasın diye anladım o zaman. bunu yaptığında eski apcnin adı odayla birlikte değişiyor mu? ayrıca gereksiz kafa karıştırıcılığı aslında ama bir şey demeyeceğim. adını keep_apc koysan daha iyi olur

Reply:
apcnin bulundugu area zaten mapping subsysteminde değiştirildigi icin apc initlendiginde o area ismiyle başlıyor